Answers.md

```diff
@@ -0,0 +1,5 @@
+1 - Semantic html is a way of marking up the html so that you can easily understand the information that you are putting on a webpage. It can be very helpful when you are setting up the organization and layout of page. 
+2 - display:block will always start on a new line and take up the entire width of the page. display:inline elements will not start on a new line, and these elements are contained in the space of their opening and closing tags.
```

Comment:
You might also add that the dimensions of inline elements cannot be altered while the dimensions of block elements can.